packages/babel-plugin/src/types.tsx

```diff
@@ -75,6 +75,11 @@ export interface Metadata {
    * Path of a parent node.
    */
   parentPath: NodePath<any>;
+
+  /**
+   * Path of the own node.
+   */
+  ownPath?: NodePath<any>;
```

Comment:
nit: `ownPath` -> `path`?

Reply:
I have kept it as `ownPath` for now. Added detailed description for it.